apps/central-scan/frontend/src/screens/scan_ballots_screen.tsx

```diff
@@ -247,12 +244,12 @@ export function ScanBallotsScreen({
           <Modal
             title={
               <span>
-                <Icons.Warning color="warning" /> Backup Required
+                <Icons.Warning color="warning" /> CVR Backup Required
               </span>
             }
             content={
               <P>
-                Go to <Font weight="semiBold">Settings</Font> and save a backup
+                Click <Font weight="semiBold">Save CVRs</Font> to save a backup
```

Comment:
@adghayes tagging in case you want to review the copy on this pr

Reply:
Suggestion, to match the other screen, "You must save CVRs before you can delete all batches." 

The button is right on the screen so I think we don't need to indicate what to click, and that pattern is not super common in `vxsuite` anyway (although I could see an argument for it).

Reply:
Agreed, updated